### DISCUSSION TITLE
Add stopdetection function

### DISCUSSION THREAD

**Discussion by tanersekmen** [networkx/networkx#6511] [Q&A] [answered]
Hi,

You can add a stop detection function to increase usage of your framework in companies/person code.
We need this function to use. I can recommend developing this function in your framework.

**Comment by MridulS** [comment, marked as answer]
What does stopdetection mean? Could you please give some more details?

**Reply by tanersekmen** [reply]
@MridulS  For example, when I want to detect a stop GPS data on the map, I can find with your function like movingpandas framework and stop detection function from this library. Did I explain exactly what I'm suggesting?

**Reply by MridulS** [reply]
Sorry, not really. Could you give a more detailed example? And what does networkx need to implement here?

**Reply by tanersekmen** [reply]
Okay, Let's give an example. TrajectoryStopDetector function is useful. If you add a stopdetector function in networkx, the number of users can increase.

**Reply by MridulS** [reply]
Alright, this is the package https://github.com/movingpandas/movingpandas right? I have no idea how this stopdetector function would work inside networkx but feel free to go ahead and open an issue/PR if you have any ideas how to implement this in networkx?

**Reply by tanersekmen** [reply]
Yes exactly, thank you so much for replying
